front/src/components/changePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { Box, Button, FormControl, FormLabel, Heading, Input, useColorMode } from '@chakra-ui/core';
+
+const FormFields = [
+  {
+    name: 'Old Passowrd',
+    labelName: 'oldPassword',
+  },
+  {
+    name: 'New Password',
+    labelName: 'newPassword',
+  },
+  {
+    name: 'Confirm New Password',
+    labelName: 'newPasswordConfirmation',
+  },
+];
+
+const ChangePassword = () => {
```

Comment:
Could we change this name to ChangePasswordContainer? Because we might move this in a modal or something else later so this will be more helpful

Reply:
Sure